modules/con-vuln-sources.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+// * cloud_service/acscs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="con-vuln-sources_{context}"]
+= Vulnerability data sources
+
+{product-title-short} uses the following vulnerability data sources:
+
+[id="red-hat-sources_{context}"]
+== Red{nbsp}Hat sources
+
+Sources for vulnerabilities depend on the scanner type and {product-title-short} version used.
+
+For {product-title-short} versions 4.5 and earlier, link:https://access.redhat.com/security/data/oval/v2/[Red{nbsp}Hat OVAL] v2 files are used. Additionally, in versions later than 4.6, if the StackRox scanner is used for vulnerability scanning, OVAL v2 is used as the source.
+
+Beginning with {product-title-short} version 4.6, https://access.redhat.com/security/data/csaf/v2/vex/[Red{nbsp}Hat VEX] is used. This source provides vulnerability data in link:https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#45-profile-5-vex[Vulnerability Exploitability eXchange(VEX)] format. VEX significantly shortens the time needed for the initial loading of vulnerability data, and VEX data takes up less storage space than OVAL data.
+
+OVAL vulnerability data is more advisory based, while VEX data is more CVE based. One advisory can have multiple CVEs associated with it. Therefore, {product-title-short} can list different numbers of vulnerabilities when you are scanning with a {product-title-short} version that uses OVAL, such as {product-title-short} version 4.5, and a version that uses VEX, such as version 4.6. Additionally, VEX data does not include vulnerabilities with a status of "under investigation", while these vulnerabilities are included in OVAL data.
+
+For more information about Red Hat security data, including information about the use of OVAL, Common Security Advisory Framework Version 2.0 (CSAF), and VEX, see link:https://www.redhat.com/en/blog/future-red-hat-security-data[The future of Red Hat security data].
+
+//is this still applicable?
+The https://access.redhat.com/security/data/metrics/cvemap.xml[Red{nbsp}Hat CVE Map]: This is used for images which appear in the link:https://catalog.redhat.com/software/containers/explore[Red{nbsp}Hat Container Catalog].
+
+[id="other-vuln-sources_{context}"]
+== Additional vulnerability sources
```

Comment:
related to suggestion above: should we split this into scanner v2 and v4 (even though some sources are the same, some are new in v4 and some have changed between versions)?